### ISSUE TITLE
gather-logs bundle should include information about any enabled beta features

### ISSUE THREAD

**Issue by teknofire** [chef/automate#1997]
## Describe the bug
When a user has enabled beta features in Automate that information should be captured as part of the gather-log bundle.  For example, when the IAMv2 or IAMv2.1 betas are enabled it's extremely useful information to have when debugging login and authentication issues for customers.

## To Reproduce
* enable IAMv2 with `chef-automate iam upgrade-to-v2`
* Run `chef-automate gather-logs` and check for information about IAMv2 or potentially any other beta feature.

## Expected behavior
The gather-log bundle should include information about beta features that have been enabled by the user.

## Versions (please complete the following information):
- OS: N/A
- Browser: N/A
- Automate Build Number: latest

**Comment by susanev** [comment]
idk if its helpful but you can check the iam version with `chef-automate iam version`

**Comment by danielsdeleo** [comment]
This issue has become (at least temporarily) obsolete, as all feature-flagged backend features have been GA-released, and there are no backend feature flags left to log.

If we wanted to modify gather logs to report on possible future feature flags, we would need to know how we could detect the existence and status of the flags, but Automate does not have infrastructure to support that. The various backend feature flags we have implemented in the past were implemented in different ways suited to the requirements of the feature being flagged. Implementing a standardized framework for feature flags would be great but adds a lot of scope to work that doesn't have any benefit at the moment.

I'm closing this ticket but I would agree that future feature flag implementations should have their state included in the gather logs bundle.
